Source/WebKit/UIProcess/API/glib/WebKitProtocolHandler.cpp

```diff
@@ -286,6 +289,10 @@ void WebKitProtocolHandler::handleGPU(WebKitURISchemeRequest* request)
     auto displayObject = JSON::Object::create();
     startTable("Display Information"_s);
 
+    auto& page = webkitURISchemeRequestGetWebPage(request);
+    auto displayID = page.displayID();
+    addTableRow(displayObject, "ID"_s, String::number(displayID.value_or(0)));
```

Comment:
Only `ID` as the title? How about using `Display Identifier`?

```suggestion
    addTableRow(displayObject, "Display Identifier"_s, String::number(displayID.value_or(0)));
```

Reply:
This is right below the section "Display information" so Display is omitted in all its fields. Maybe just Identifier?

Reply:
Yes, `Identifier` is perfect 👌🏼